llvm/lib/CodeGen/StackMaps.cpp

```diff
@@ -305,12 +307,13 @@ StackMaps::parseOperand(MachineInstr::const_mop_iterator MOI,
     // and during deoptimisation we need to make sure we write the value back to
     // each one of them. Note, that the stackmap may track either of %rbx or
     // %rcx, resulting in different ways below to retrieve the mappings.
-    int ExtraReg = 0;
+    //int ExtraReg = 0;
```

Comment:
commented.

Reply:
Fixed in [9d435de](https://github.com/ykjit/ykllvm/pull/204/commits/9d435de80dd42e600e4a249b72a4dc3615c26901)